src/pretix/presale/templates/pretixpresale/base.html

```diff
@@ -33,14 +33,14 @@
     {% endcompress %}
     <meta name="referrer" content="origin">
     {{ html_head|safe }}
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0, user-scalable=0">
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=0">
     <link rel="icon" href="{% static "pretixbase/img/favicon.ico" %}">
     {% block custom_header %}{% endblock %}
 </head>
 <body class="nojs" data-locale="{{ request.LANGUAGE_CODE }}" data-now="{% now "U.u" %}" data-datetimeformat="{{ js_datetime_format }}" data-timeformat="{{ js_time_format }}" data-dateformat="{{ js_date_format }}" data-datetimelocale="{{ js_locale }}">
 {% block above %}
 {% endblock %}
-<div class="container">
+<div class="container" role="main">
```

Comment:
Is this correct? As I understand it, `role="main"` should contain only the actual main content of the document, allowing users to skip repetetive content that appears on every page. For pretix, this would include at least the event header, I'd say.

Reply:
Hmm... :thinking: I don't know much about this so I'll do some research and get back to you. :)

Reply:
Sorry about the delay. 
Yeah, you are right. However, I tried to add this to a more relevant part (say, the form) but that cannot be done that way. So I ended up deciding to remove this attribute altogether. 
Now, sad story: my branch was stale and I messed up refreshing it :disappointed: 

So here's the real question: considering I never discussed this proposal with any of you before, do you think it's valuable to add? If so, I will see if I can figure out a way of recovering from my mess or might end up closing this PR and sending a new cleaner one. On the other hand, if you think it's not the time, I will be super ok with that too (and we can close this PR).